### PR TITLE
initial server commit

### PR DESCRIPTION
**Pull request by LBWright**

### CODE REVIEW

**Comment by LBWright** [comment]
@johnoro this is the initial server setup to be pulled into master

**Review by johnoro — APPROVED** [review]
Looks good to me. In the future I'll try to wait for an approval from a teammate since they're able to be more involved and will probably help with avoiding merge conflicts more.

**Pull request merged**
